test/e2e/fixture-builder.js

```diff
@@ -0,0 +1,647 @@
+class Fixture {
+  constructor() {
+    this.data = {
+      AddressBookController: {},
+      AlertController: {},
+      AnnouncementController: {
+        announcements: {
```

Comment:
I have a couple of suggestions for the announcements part. Since it's pretty hard to tell what each number represents, I would maybe add a comment on each one, with a name indicating which announcement represent. (In the code it's also hard to find, as it's displayed in the same way as here).

I've seen that there are 14 announcements. Do you think we should include all of them here? We are missing `0, 2, 12, 13 `

![image](https://user-images.githubusercontent.com/54408225/193815562-1e0d6f56-1be0-4ccc-bb42-ea04df197ab9.png)

Reply:
Yes, it should probably go all the way up to 15 for the latest notification about the merge